### ISSUE TITLE
WinAppDriver fails to find any element on Share Windows Store App Picker if using AccessibilityID but Inspect.exe sees it

### ISSUE THREAD

**Issue by db-007** [Microsoft/WinAppDriver#301]
WinAppDriver does not find elements by accessibility_id in a Windows Store Share dialog.

**STEPS:**
I have an application that lets me share log files

1. When I click on a button Share dialog pops up giving me windows store apps I can use to share my log file such as Mail, OneNote, etc 
2. Any element having AutomationId, including Mail Windows Store App can not by found by AutomationId but can be found by Name (see attached image)
3. However, Inspect.exe app sees both Name and AutomationId (as per attached image)

Below is code sample using Name that works and code sample using AccessibilityID that fails.

**CODE THAT FINDS ELEMENT USING NAME**
```ruby
When /^I see external element "([^\"]*)"$/ do |name|
		sleep(2.0)
		desktop_session_setup()
		item = $DesktopSession.find_elements(:name, name)[0]
		desktop_session_end()
	
		unless item != nil
			fail("External element with #{name} not shown.")
		end
end
```
**CODE THAT FAILS TO FIND ELEMENT USING ACCESSIBILITYID**
```ruby
When /^I see external "([^\"]*)" by id$/ do |accessibility_id|
    sleep(2.0)
    
    desktop_session_setup()
    item = $DesktopSession.find_elements(:accessibility_id, accessibility_id)[0]
    desktop_session_end()
	
    unless item.displayed?
      fail("Item with id #{accessibility_id} not found.")
    end
end
```
![winappdriver](https://user-images.githubusercontent.com/17257073/32683100-8d2186ac-c62d-11e7-9eb1-f7e053444790.png)

![sharedialog](https://user-images.githubusercontent.com/17257073/32682772-b5cb3b04-c62b-11e7-9ba3-9532671e8c31.png)

**Comment by timotiusmargo** [comment]
Hi @db-007,

Thanks for the detail annotated screenshots that provide clear context of the issue. Would you be able to provide the log from `WinAppDriver.exe` for both find elements commands above? We would like to see the JSON wire protocol command generated and its payload from your script. E.g.

```
==========================================
POST /session/D4348D8A-BD5C-41F0-A216-9B0EB0557378/elements HTTP/1.1
Accept: application/json, image/png
Content-Length: 53
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723

{"using":"accessibility id","value":"AlarmPivotItem"}
HTTP/1.1 200 OK
Content-Length: 100
Content-Type: application/json

{"sessionId":"D4348D8A-BD5C-41F0-A216-9B0EB0557378","status":0,"value":[{"ELEMENT":"42.71938.3.3"}]}
```

**Comment by db-007** [comment]
@timotiusmargo how do I get you the log you need?

**Comment by timotiusmargo** [comment]
Typically, `WinAppDriver.exe` can run as a standalone program running as a console application logging all requests and responses in your machine under test. If you run other client on top such as Appium, the logs are normally embedded within the client log.

**Comment by db-007** [comment]
Sorry, but I am unable to find any logs

**Comment by AmrutaDJoshi** [comment]
I am facing similar issue
==========================================
POST /session/DD0375E5-70D8-4EC4-8D92-908FF9FA88CB/element HTTP/1.1
Accept: application/json, image/png
Content-Length: 31
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723

{"using":"name","value":"ANPS"}
HTTP/1.1 404 Not Found
Content-Length: 139
Content-Type: application/json

{"status":7,"value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters."}}


==========================================
DELETE /session/DD0375E5-70D8-4EC4-8D92-908FF9FA88CB HTTP/1.1
Accept: application/json, image/png
Host: 127.0.0.1:4723


HTTP/1.1 200 OK
Content-Length: 12
Content-Type: application/json

{"status":0}

**Comment by timijntema** [comment]
I seem to have a similar issue. I use inspect to find the AutomationId and then use FindElementByAccessibilityID() to find the element. In my case this fails when the app is a self build C# GUI. It is not just one element. I commented out the element it could not find and ran the tests to see if it could find the next, but that did not work either.

The code I use in C# to create the test:
``` C#
// Launch the app
AppiumOptions options = new AppiumOptions();
options.AddAdditionalCapability("app", "C:\\path\\to\\app.exe");
options.AddAdditionalCapability("appArguments", "\"some-argument\"");
try
{
	this.appSession = new WindowsDriver<WindowsElement>(new Uri("http://127.0.0.1:4723"), options);
}
catch (ArgumentNullException error)
{
	Console.WriteLine(error.Message);
}
catch (UriFormatException error)
{
	Console.WriteLine(error.Message);
}

// ResizeWindow
var maximizeButton = this.appSession.FindElementByAccessibilityId("Maximize-Restore");
maximizeButton.Click();
```

The console logs from WinAppDriver:
```
==========================================
POST /session HTTP/1.1
Accept: application/json, image/png
Connection: Keep-Alive
Content-Length: 249
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723
User-Agent: selenium/3.141.0 (.net windows)

{"desiredCapabilities":{"app":"C:\\path\\to\\app.exe","appArguments":"\"some-argument\"","platformName":"Windows"},"capabilities":{"firstMatch":[{"platformName":"Windows"}]}}
HTTP/1.1 200 OK
Content-Length: 238
Content-Type: application/json




==========================================
POST /session/DEFA5321-0FB7-4992-A47F-1D0A0610412B/element HTTP/1.1
Accept: application/json, image/png
Content-Length: 55
Content-Type: application/json;charset=utf-8
Host: 127.0.0.1:4723
User-Agent: selenium/3.141.0 (.net windows)

{"using":"accessibility id","value":"Maximize-Restore"}
HTTP/1.1 404 Not Found
Content-Length: 139
Content-Type: application/json

{"status":7,"value":{"error":"no such element","message":"An element could not be located on the page using the given search parameters."}}
```

**Comment by timijntema** [comment]
I installed V1.1.1 for the WinAppDriver and the issue seems to be resolved.
